mixer/adapter/cloudwatch/README.md

```diff
@@ -0,0 +1,56 @@
+# Amazon CloudWatch Adapter
+
+This is an Istio Mixer adapter for integration with Amazon CloudWatch. Currently it supports sending metrics to CloudWatch.
+
+## Configuration
+
+To push metrics to CloudWatch using the adapter you need create an IAM user that has permissions to call cloudwatch APIs. The credentials for the user need to be available on the instance the adapter is running on (see [AWS docs](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html)).
+
+To activate the CloudWatch adapter, operators need to provide configuration for the
+[cloudwatch adapter](https://istio.io/docs/reference/config/adapters/cloudwatch.html).
+
+The handler configuration must contain the same metrics as the instance configuration. The metrics specified in both instance and handler configurations will be sent to CloudWatch.
+
+## Example configuration
+
+```yaml
+# instance configuration for template 'metric'
+apiVersion: "config.istio.io/v1alpha2"
+kind: metric
+metadata:
+ name: cloudwatchrequestcount
```

Comment:
question: is it possible for the cloudwatch adapter to ingest the existing `requestcount.metric` (and similar) instances?

ideally, that would be how this integration would work. is the issue label renaming?